packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -1219,7 +1215,7 @@ class _TableRowState<T> extends State<TableRow<T>>
   Widget build(BuildContext context) {
     final row = tableRowFor(
       context,
-      onPressed: () => widget.onPressed(widget.node),
+      onPressed: () => widget.onPressed!(widget.node),
```

Comment:
onPressed can be null. Do a null check here:
 `widget.onPressed != null ? widget.onPressed!(widget.node) : null;`

Reply:
done